package.json

```diff
@@ -41,7 +41,6 @@
     "ember-cli-moment-shim": "0.6.1",
     "ember-cli-qunit": "1.0.1",
     "ember-cli-uglify": "^1.0.1",
-    "ember-data": "~2.0.0",
```

Comment:
I think as of right now its needed in both package.json and bower.json. This change broke the demo app so I had to revert it.